composer/callbacks/system_metrics_monitor.py

```diff
@@ -83,3 +136,17 @@ def compute_system_metrics(self):
         for k, v in network_usage.items():
             system_metrics[f'network_{k}'] = v
         return system_metrics
+
+    def compute_min_max_metrics(self, all_metrics, model_device):
```

Comment:
```suggestion
    def compute_gpu_min_max_metrics(self, all_metrics, model_device):
```
Can we rename to be a bit more explicit since it ignores the CPU ones?